docs/source/normalizing_data/sources.rst

```diff
@@ -0,0 +1,22 @@
+Sources
+=======
+
```

Comment:
I think it might make more sense to move the Gene record summary over to this page, and put GeneDescriptor on the normalization page

Reply:
Or even having sections for search, normalize, and normalize_unmerge?